plugins/org.jkiss.dbeaver.ui.editors.connection/src/org/jkiss/dbeaver/ui/dialogs/driver/DriverEditDialog.java

```diff
@@ -628,6 +630,8 @@ private void addMavenArtifact() {
             libraries.addAll(fd.getArtifacts());
             changeLibContent();
         }
+
+        driver.resetDriverInstance();
```

Comment:
Reset before save?
I think we can reset instance only if model changes (DriverDescriptor)